pentaho-js/src/test/javascript/cdf/components/TableComponent-spec.js

```diff
@@ -196,5 +196,27 @@ define([
       dashboard.update(tableComponent);
     });
 
+    /**
+     * ## The Table Component # can update when server side is true
+     */
+    it("can update when server side is true", function(done) {
```

Comment:
I am fine with making the test just test the new code. This will allow us to write more tests that are unit tests and eventually cover the whole component. I would prefer to do this instead of perpetuating functional unit tests incorrectly.